charts/prometheus-postgres-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.10.1"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 3.1.5
+version: 3.1.6
```

Comment:
new feature should raise minor version

Reply:
appVersion: "0.10.2"
version: 3.1.6 
is this correct ?

Reply:
version field describes version of the chart.
appversion describes version of the app / docker image (if there is a newer docker image available you might want to update that too).

minor version change would be: 3.2.6 for the version field.
See: https://semver.org/